boa_cli/src/main.rs

```diff
@@ -362,8 +364,12 @@ impl Highlighter for LineHighlighter {
                         }
                         _ => cap.as_str().to_string(),
                     }
-                } else if let Some(cap) = caps.name("op") {
+                } else if let Some(cap) = caps.name("string_double_quote") {
+                    cap.as_str().green().to_string()
+                } else if let Some(cap) = caps.name("string_single_quote") {
                     cap.as_str().green().to_string()
+                } else if let Some(cap) = caps.name("op") {
+                    cap.as_str().truecolor(214, 95, 26).to_string()
```

Comment:
Would it be better to have the colours defined as constants at the top of the file?